src/styles/utils/functions.scss

```diff
@@ -0,0 +1,3 @@
+@function center-stopwatch-elements($size) {
+  @return ($stopwatch-size * 0.5) - ($size * 0.5);
```

Comment:
There seems to be an undefined variable `$stopwatch-size` used in the function `center-stopwatch-elements`. Make sure all variables are defined within the scope of their usage or are imported from other files where they are defined. This is crucial for the function to work correctly.